src/set/set.service.ts

```diff
@@ -119,6 +129,10 @@ export class SetService {
 
         if (user.role !== Role.ADMIN) queryMatch.createdBy = user.userId;
 
+        if (updateSetDto.name) {
+            updateSetDto.slug = slugged(updateSetDto.name);
+        }
+        console.log(updateSetDto.slug);
```

Comment:
didnt see this one when checking...please remove

Reply:
Removed